engine/src/test/scala/cromwell/logging/WorkflowLoggerSpec.scala

```diff
@@ -5,39 +5,40 @@ import java.util.UUID
 import cromwell.CromwellTestkitSpec
 import cromwell.core.WorkflowId
 import cromwell.engine.WorkflowSourceFiles
-import cromwell.engine.backend.local.LocalBackend
-import cromwell.engine.backend.{BackendCallJobDescriptor, WorkflowDescriptorBuilder}
+import cromwell.engine.backend.local.OldStyleLocalBackend
+import cromwell.engine.backend.{OldStyleBackendCallJobDescriptor, WorkflowDescriptorBuilder}
 import cromwell.engine.workflow.BackendCallKey
 import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 import wdl4s.values.WdlValue
 
 class WorkflowLoggerSpec extends FlatSpec with Matchers with BeforeAndAfterAll with WorkflowDescriptorBuilder {
-  val testWorkflowManagerSystem = new CromwellTestkitSpec.TestWorkflowManagerSystem()
-  override implicit val actorSystem = testWorkflowManagerSystem.actorSystem
+  lazy val testWorkflowManagerSystem = new CromwellTestkitSpec.TestWorkflowManagerSystem()
+  override implicit lazy val actorSystem = testWorkflowManagerSystem.actorSystem
```

Comment:
Because otherwise the test constructor itself starts up a whole bunch of OldStyle stuff. I may have got a tiny bit overzealous with the laziness because I didn't want to test every line for importance... (shrug).